Add Cloud Scale Inc as pro services

nandita-cloudscaleinc opened the pull request:
Please provide a description for what this PR is for.

Add Cloud Scale® Inc as pro service provider for ISTIO service mesh

And to help us figure out who should review this PR, please put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

istio-testing commented:
Hi @nandita-cloudscaleinc. Thanks for your PR.

I'm waiting for a [istio](https://github.com/orgs/istio/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/istio/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=istio%2Fistio.io).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

ericvn commented:
/ok-to-test

nandita-cloudscaleinc commented:
Can someone pls review this. Thanks !

craigbox commented:
we need to close out https://github.com/istio/istio.io/issues/11240 first I think.

I would also recommend you remove all the "® Inc" (or only include it once) please.

nandita-cloudscaleinc commented:
> we need to close out #11240 first I think.
> 
> I would also recommend you remove all the "® Inc" (or only include it once) please.

Thanks Craig. Addressed the comments and kept it only at the required places!

nandita-cloudscaleinc commented:
@ericvn can you pls review this request

nandita-cloudscaleinc commented on data/companies.yml:
```diff
@@ -292,6 +292,13 @@ pro_services:
+    description: "Cloud Scale® Inc experts assess your current applications, runtime deployments, service maps and use-cases, find the right mix of cloud platforms for your workloads, Identify gaps in technology & skills, and prepare a success plan – The ones that will help you accomplish your business objectives"
+    details:
+      - "Cloud Scale platform and service mesh experts puts distribution & management in the palm of your hands. From productivity to continuity, from security to remote monitoring & management – our growing portfolio of services has got you covered. "
+      - "Cloud consulting and Istio service mesh experts by Cloud Scale helps in solving all of the needs by selecting the right platforms, cloud solutions and platform needs that integrate smoothly with legacy & next-generation technology’s and deliver optimal return on technology investment (TCO)"      
```
addressed the review comments. Thanks for the feedback

nandita-cloudscaleinc commented:
Can this PR be pls reviewed . Thanks!

cc - @ericvn @craigbox

nandita-cloudscaleinc commented:
Gentle Reminder. Can someone pls review this. Thanks

craigbox commented on data/companies.yml:
```diff
@@ -292,6 +292,13 @@ pro_services:
+  - name: "Cloud Scale® Inc"
+    logo: "/logos/cloudscaleinc.png"
+    url: "https://cloudscaleinc.com/"
+    description: "Cloud Scale® Inc experts assess your current applications, runtime deployments, service maps and use-cases, find the right mix of cloud platforms for your workloads, identify gaps in technology & skills, and prepare a success plan – The ones that will help you accomplish your business objectives."
```
```suggestion
    description: "Cloud Scale® Inc experts assess your current applications, runtime deployments, service maps and use-cases, find the right mix of cloud platforms for your workloads, identify gaps in technology & skills, and prepare a success plan."
```

I think it's too much longer than the others.  If this LGTM, I can merge.

nandita-cloudscaleinc commented:
addressed review comments. Please let me know if there's any other feedback!
